frontend/src/setupProxy.js

```diff
@@ -0,0 +1,12 @@
+const proxy = require('http-proxy-middleware');
+
+// @ts-ignore
+module.exports = function (app) {
```

Comment:
Is this used just in the development environment? Do we need to set the `target` used by the function below on a environment variable?

Reply:
This is just for development. It will proxy the backend APIs so you don't need to bother with CORS or OAuth, you can just use session auth.

Reply:
This is just for development. It will proxy the backend APIs so you don't need to bother with CORS or OAuth, you can just use session auth.